lib/stencil-bundle.js

```diff
@@ -354,14 +379,16 @@ function bundleThemeFiles(archive, themePath, configuration) {
  * Archive all generated files (ex. parsed files)
  * @param {Archive} archive
  * @param {Object} taskResults
+ * @returns {Array} 
  */
-function bundleParsedFiles(archive, taskResults) {
+Bundle.prototype.bundleParsedFiles=function (archive, taskResults) {
```

Comment:
There shouldn't be any need to make this part of the Bundle object. The changes you're making are purely functional and don't need access to the Bundle object.

Reply:
That change was in order to be able to unit test it.

Reply:
Ok, I recommend splitting this change out into a separate PR so we can discuss other potential strategies for that. It's also an unrelated change, and it's best to do them separately.

Reply:
Used a fresh copy of stencil-cli and made changes to reflect only the dist folder check.  Will make another pull request with the template check after version updating on the font-display stuff.